provider/mediaconvert/testing.go

```diff
@@ -0,0 +1,102 @@
+package mediaconvert
```

Comment:
Just need to name this file with a ``_test`` suffix (something like ``fake_client_test.go``, maybe?). Just to make sure it doesn't get compiled with the production binaries (I know the linker will likely eliminate it as dead code, but it's good to be safe!)

Reply:
Will do

Reply:
done